modules/router-store/src/router_selectors.ts

```diff
@@ -59,6 +58,10 @@ export function getSelectors<V>(
     selectRouterState,
     (routerState) => routerState && routerState.url
   );
+  const selectTitle = createSelector(
```

Comment:
I would think this would use `selectCurrentRoute` because the `title` can be defined on any route config.

Reply:
Done